src/Command/ResetAppCommand.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use Symfony\Component\Console\Attribute\AsCommand;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Process\Process;
+
+#[AsCommand(name: 'app:reset')]
+class ResetAppCommand extends Command
+{
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        // Install dependencies
+        $composerInstall = new Process(['composer', 'install', '-n']);
+        $composerInstall->mustRun();
+
+        if (!$composerInstall->isSuccessful()) {
+            echo $composerInstall->getOutput();
+            return Command::FAILURE;
+        }
+
+        // Remove existing tables
+        $schemaDrop = new Process(['bin/console', 'doctrine:schema:drop', '--force']);
```

Comment:
https://symfony.com/doc/6.4/console/calling_commands.html

Reply:
Thanks, I just refactored the doctrine command to use ArrayInput